projects/plugins/jetpack/CHANGELOG.md

```diff
@@ -2,6 +2,41 @@
 
 ### This is a list detailing changes for all Jetpack releases.
 
+## 14.0-a.3 - 2024-10-14
+### Enhancements
+- Newsletter: Update the default "reply to" setting value from not allowed to replies being posted as public comments. [#39657]
+- Social: Add Bluesky to social previews. [#39659]
+
+### Improved compatibility
+- Image CDN: URL encode image path parts for RSS feed compatibility. [#39560]
+- Related Posts: Allow Related Posts on non-post CPTs where the block is already able to be used. [#39730]
+
+### Bug fixes
+- Blocks: Fix rendering of the goodreads block to avoid PHP warnings caused by missing attributes. [#39713]
+- Blocks: Render the slideshow block correctly inside an iframe editor. [#39551]
+- Contact Form: Ensure that submitted forms can only be accessed by logged in users allowed to view form submissions. [#39759]
```

Comment:
Do you think we should / could create a 13.9.1 entry in the changelog, with only this change in there, to match the release in the plugin directory? If we don't, I worry the 13.9.1 release will never make it to this changelog file.

Or maybe we should do that in a follow-up PR?

Reply:
I'd do it in a follow-up PR, I think.

Reply:
Done.